extscheduler/resources/schemas/dbscripts/sqlserver/extscheduler-20.936-20.937.sql

```diff
@@ -0,0 +1,13 @@
+EXEC core.fn_dropifexists 'TempCoV19Final', 'extScheduler', 'TABLE', NULL;
+    GO
+
+CREATE TABLE [extscheduler].[TempCoV19Final](
+    [container] [uniqueidentifier] NOT NULL,
+    [UserName] [nvarchar](4000) NULL,
+    [UserID] [int] NULL,
+    [EventID] [int] NULL,
+    [StartDate] [smalldatetime] NULL,
+    [EmployeeID] [nvarchar](4000) NULL,
+    [Created] [smalldatetime] NULL
+    ) ON [PRIMARY]
+    GO
```

Comment:
Do we also need to update the schema version to 20.937 in ExtSchedulerModule? currently it is at 20.603.

Reply:
Yes, this file had a merge conflict which I've now resolved and bumped the version number.